app/services/organizations/create_service.rb

```diff
@@ -24,7 +24,8 @@ def signal(args)
     ActiveRecord::Base.transaction do
       create_organization(
         args[:organization][:name],
-        args[:organization][:slug]
+        args[:organization][:slug],
+        args[:organization][:email]
       )
       create_location(
```

Comment:
I think we can get rid of the location, too right? It used to be required, but we are not validating for it.